Automattic/vip-design-system
# Valet Theme Updater — Token Transformer

## Description

Introducing a way to have the VIP Design System (Figma files) integrated with all our React components 🥳 . The long explanation is described in the README.md file changes in this Pull Request.

The long history short is: We now can read all theming information we need from a tokens JSON file and parse it so the React theme can be read and used in the components system.

New npm script command added:

```bash
npm run theme-update
```

## Checklist


## Steps to Test

1. Pull down PR.
2. `npm run theme-update`.
3. Check the `src/generated/valet-theme.json` file
4. Run `npm run dev`
5. See the new colors are applied to all components
